### ISSUE TITLE
Library is not open to debugging/extension

### ISSUE THREAD

**Issue by SteveBallCS** [NISystemsEngineering/LabVIEW-Modbus-API#12]
This library has locked VIs.  Please unlock them or provide a password.

**Comment by bboyz1224** [comment]
I agree. NI should definitely unlock the VIs to let everyone optimizes this library.

**Comment by OlivierJourdan** [comment]
I'm maybe wrong, but I'm not sure the VIs are locked in the repo. They are only locked in the VIP.
If you clone the repo, you can access the entire code.

**Comment by bboyz1224** [comment]
> I'm maybe wrong, but I'm not sure the VIs are locked in the repo. They are only locked in the VIP. If you clone the repo, you can access the entire code.

Thanks Olivier,
I'll give it a look and see if it's locked in this repo source.
